Atomic/syscontainers.py

```diff
@@ -306,6 +309,23 @@ def _do_checkout(self, repo, name, img, upgrade, values, destination, unitfileou
         if rev is None:
             raise ValueError("Image %s not found" % img)
 
+        if not upgrade and not extract_only:
```

Comment:
we can use `--set` on `update --container` as well.  Can we drop the `not upgrade` condition here?